XLS-30/README.md

```diff
@@ -0,0 +1,1255 @@
+<pre>    
+  Title:        <b>Automated Market Maker on XRPL</b>
+  Revision:     <b>4</b> (2023-08-08)
+
+<hr>  Author:       <a href="mailto:amalhotra@ripple.com">Aanchal Malhotra</a>
+                <a href="mailto:david@ripple.com">David J. Schwartz</a>
+  Affiliation:  <a href="https://ripple.com">Ripple</a>
+</pre>
+# Automated Market Maker on XRP Ledger
+
+### Abstract
+The XRPL decentralized exchange (DeX) currently provides liquidity exclusively by manual market making and order books. This proposal adds non-custodial automated market maker (AMM) as a native feature to the XRPL DeX in a way that provides increased returns to those who provide liquidity for the AMM and minimizes the risk of losses due to volatility.
```

Comment:
```suggestion
The XRPL decentralized exchange (DEX) currently provides liquidity exclusively by manual market making and order books. This proposal adds non-custodial automated market maker (AMM) as a native feature to the XRPL DEX in a way that provides increased returns to those who provide liquidity for the AMM and minimizes the risk of losses due to volatility.
```
For clarity, it should be capitalized as DEX, as the E stands for Exchange.

![Untitled](https://github.com/XRPLF/XRPL-Standards/assets/81505/ff87686c-1ab5-4db7-a322-01a37d37807d)
![Untitled 2](https://github.com/XRPLF/XRPL-Standards/assets/81505/30548054-21d0-47e5-9905-66a293ffcdb8)
![Untitled 3](https://github.com/XRPLF/XRPL-Standards/assets/81505/1bd6d66c-e17a-4987-84a3-85fe0872fc97)